core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -80,11 +82,11 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
   import dataSource.dataAccess.driver.api._
 
   def CreateWorkspace(workspace: WorkspaceRequest, parentSpan: Span = null) = createWorkspace(workspace, parentSpan)
-  def GetWorkspace(workspaceName: WorkspaceName, params: WorkspaceFieldSpecs) = getWorkspace(workspaceName, params)
+  def GetWorkspace(workspaceName: WorkspaceName, params: WorkspaceFieldSpecs, parentSpan: Span = null) = getWorkspace(workspaceName, params, parentSpan)
   def DeleteWorkspace(workspaceName: WorkspaceName) = deleteWorkspace(workspaceName)
   def UpdateWorkspace(workspaceName: WorkspaceName, operations: Seq[AttributeUpdateOperation]) = updateWorkspace(workspaceName, operations)
   def UpdateLibraryAttributes(workspaceName: WorkspaceName, operations: Seq[AttributeUpdateOperation]) = updateLibraryAttributes(workspaceName, operations)
-  def ListWorkspaces(params: WorkspaceFieldSpecs) = listWorkspaces(params)
+  def ListWorkspaces(params: WorkspaceFieldSpecs, parentSpan: Span = null) = traceWithParent("listWorkspaces", parentSpan)(span => listWorkspaces(params, span))
```

Comment:
Why does listWorkspaces use `traceWithParent `, but getWorkspace doesn't?

Reply:
listWorkspaces was vestigial, removed this level of tracing